src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -412,10 +341,10 @@ export default class PeopleAndRoles extends Mixins(CommonMixin, DateMixin, OrgPe
       this.isEntityBcCompany || this.isEntityBenefitCompany || this.isEntityBcUlcCompany ||
       this.isEntityContinueIn || this.isEntityBenContinueIn || this.isEntityUlcContinueIn
     ) {
-      return this.hasRole(RoleTypes.DIRECTOR, 1, CompareModes.AT_LEAST)
+      return this.hasRole(RoleTypes.DIRECTOR, 0, CompareModes.AT_LEAST)
```

Comment:
Why not just return True?

Or could this computed be deleted altogether? (And also delete haveMajorityDirectorsInCanada?)